spec/spec.md

```diff
@@ -436,6 +449,8 @@ where:
 
 This method attempts to renew the specified `auth_token`. The meaning of renew is an implementation detail of the wallet endpoint; it may return the same token, issue a new token, or refuse to renew the token. If the result is `ERROR_AUTHORIZATION_FAILED`, the token could not be renewed. A new token will need to be requested with the [`authorize`](#authorize) method. If a new `auth_token` is returned, the dapp endpoint should assume that the old `auth_token` is no longer valid and discard it.
 
+In addition to the reauthorized auth token, the wallet endpoint may optionally return updated values for `public_key` and `wallet_uri_base`, overriding any previously provided values. This allows a wallet endpoint to modify the set of authorized accounts for an already-issued auth token, without requiring the dapp endpoint to restart the authorization process.
+
```

Comment:
Wallets have a few options here:
1. The new parameters to reauthorize can be used if the wallet wishes to automatically add or remove account access for previously issued auth tokens. Internally, it would set a flag on all modified auth tokens indicating that they require reauthorization. The next time they are attempted to be used, the dapp would get an `ERROR_REAUTHORIZE` result, would perform reauthorization, and receive updated accounts.
2. If a wallet wishes to force dapps to go through a full authorization process again, they can just invalidate all auth tokens in their internal store, and dapps will get an `ERROR_AUTHORIZATION_FAILED` result. At this point, the dapp should discard the cached auth token and re-run authorization
3. Some hybrid of the two - use `ERROR_REAUTHORIZE` to remove accounts, and `ERROR_AUTHORIZATION_FAILED` to add accounts (by requiring a new authorization.